### ISSUE TITLE
A possible solution to the Fallout4.exe not exiting properly

### ISSUE THREAD

**Issue by archtrtd1** [TanninOne/modorganizer#894]
I may have found the reason Fallout4.exe wasn't shutting down properly!  For reference, I am running Mod Organizer v2.0.7, Windows 7 64 bit, and Fallout 4 v1.6.9.0.

I am one of the people who had the Fallout4.exe continue to run in the background after exiting to desktop.  After having some puzzling ctds I decided to add all of the associated executables (Fallout4.exe, F4SE.exe, ModOrganizer.exe, etc) to the "Excluded Processes" list in Microsoft Security Essentials.  Suddenly I was able to exit out of the game completely (and ended my ctds, as well).  I think Security Essentials was causing some interference with Mod Organizer.

**Comment by wolfgarr15** [comment]
I tried this, and it was not the issue for me... However, I might have found another cause. 

Are you using the Achievements Enabler mod available on the Fallout 4 Nexus? (The one that enables Steam achievements  when using mods.) I've discovered that as soon as I remove the DLL loader that comes with this mod, Fallout 4 terminates as expected. If I install the Achievments Enabler DLL loader, Fallout 4.exe starts to hang on quit again.

If I can get someone else to confirm the above, I will report this to both Tannin and the author of the Achievements Enabler.

**Comment by Adun32** [comment]
^Do you have to completely uninstall achievements mod?

I tried turning it off, but it didn't change anything. Does the .dll actually load and can be seen in the task manager when the mod is turned on? Because I don't see it either way.

In any case, same issue as presented by both of you. Haven't found a solution yet.

**Comment by wolfgarr15** [comment]
Yes, you must uninstall the mod, particularly the XAudio DLL. The audio DLL is the external DLL loader. Removing the DLL loader is what fixed the issue for me.

**Comment by archtrtd1** [comment]
Well, I have to inform everyone that Fallout4.exe will not properly shut down again.  I started adding a few workshop related mods to my game and discovered that "Place Everywhere" (a F4SE plugin .dll) is now causing this problem again.  

However, Looks Menu, which also utilizes a F4SE .dll, does not cause me any exiting problems.

I have posted a "bug report" on the Place Everywhere nexus page.  I am not feeling too optimistic, though.  Mod Organizer users are in the minority and I have witnessed a few mod authors reject support because Mod Organizer is still in "Beta" (and they don't personally use it).

**Comment by archtrtd1** [comment]
Miraculously the update to Fallout 4 version 1.7.9.0, F4SE 0.2.2, and the newest Place Everywhere solved my exiting problems.  YAY!

Unfortunately a new problem popped up to replace the old one:  FO4 Edit will not launch the plugins from my heavily modded profile.  I keep getting the error:  Fatal: Error loading plugin list: <EAccessViolation: Access violation at address 6946F20C in module 'MSVCR120.dll'. Read of address 01306000>

I am going to post this problem as a new issue.
